packages/frontend/src/components/accounts/two_factor/TwoFactorVerifyModal.js

```diff
@@ -20,13 +22,45 @@ const Form = styled.form`
     width: 100%;
 `;
 
+const StyledBannerContainer = styled.div`
+    &&& {
+        div:first-child {
+            margin: 0px;
+            border-radius: 0;
+            width: 100%;
```

Comment:
nit: we want to actually override the border-radius change on 450px screen width so maybe something like this:

```suggestion
            margin: 0px;
            width: 100%;
            @media (max-width: 450px) {
                 border-radius: 4px;
            }
```